src/integrationTest/cloudformation/intellisense.test.ts

```diff
@@ -0,0 +1,116 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as path from 'path'
+import * as fs from 'fs-extra'
+import * as vscode from 'vscode'
+import * as sinon from 'sinon'
+import {
+    getCompletionItems,
+    getHoverItems,
+    getTestWorkspaceFolder,
+    MAPPING_SCHEMA_INTERVAL,
+    MAPPING_SCHEMA_TIMEOUT,
+} from '../integrationTestsUtilities'
+import { activateExtension } from '../../shared/utilities/vsCodeUtils'
+import { waitUntil } from '../../shared/utilities/timeoutUtils'
+import globals from '../../shared/extensionGlobals'
+import { VSCODE_EXTENSION_ID } from '../../shared/extensions'
+
+describe('cloudformation yaml language features', async function () {
```

Comment:
There is a lot of duplication in compared to the other test file. Can we put the common logic in one test file that iterates through different inputs?